components/post_view/post_list/post_list_virtualized.jsx

```diff
@@ -128,6 +135,7 @@ class PostList extends React.PureComponent {
             dynamicListStyle: {
                 willChange: 'transform',
             },
+            renderUnreadChannelToast: this.props.countUnread !== 0,
```

Comment:
I think there is one minor case where this would fail. The logic of `renderUnreadChannelToast` depends on the the value of `countUnread` when this component loads.  

When system goes to sleep/socket disconnect and reconnect we need to have the toast again when we sync the existing channel. Even if we account for that from what i see this kinda would be a problem as we need to address that on sync we don't take user to the `new message line` anyway. I think we should create two tickets.

1. Take user to `new message line` when socket reconnects(Come back from sleep) and new messages exist
2. After 1 we can use that as a way to show the toast.